### ISSUE TITLE
won't work outside of test

### ISSUE THREAD

**Issue by jbrandes** [resume/resume.github.com#258]
I opted-in and got my resume but it won't let me share. It says user not found.

**Comment by danielb987** [comment]
Your resume works fine for me.
http://resume.github.io/?jbrandes

**Comment by Toumash** [comment]
Yeah, for me it was to hard refresh using CTRL+SHIFT+R, should be closed
